run-pytype.py

```diff
@@ -107,34 +106,28 @@ def main(me: str, branch_url: str):
         output_file (str): output file path for the markdown summary table
         branch_url (str): _url of the branch for file links in the summary table
     """
-    never = (
-        "xcp/repository.py",
-        "tests/test_ifrename_logic.py",
-        "tests/test_xmlunwrap.py",
-    )
-    excludes = [
-        "xcp/cmd.py",
-        "xcp/net/ip.py",
-    ]
-    errors_in = excludes.copy()
-    errors_in.extend(never)
+    broken_files = []  # type: list[str]
+    xfail_files = []  # type: list[str]
+    errors_in = xfail_files.copy()
+    errors_in.extend(broken_files)
     base = [
         "pytype",
         "-k",
         "--config",
         ".github/workflows/pytype.cfg",
     ]
     command = base.copy()
-    command.extend(["--exclude", " ".join(errors_in)])
+    if errors_in:
```

Comment:
Is there any way for errors_in to have contents at this point? It's the copy of an empty list extended by an empty list

Reply:
Fixed: Like said before, these are to be configuration settings:
- Improved the code to read them from configuration files.